tests/console/yast2_clone_system.pm

```diff
@@ -38,7 +38,28 @@ sub run {
 
     # Check and upload profile for chained tests
     upload_asset "/root/autoinst.xml";
-    if (script_run 'xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml') {
+
+    unless (is_opensuse) {
+        my $sle15_repo;
+        if (is_sle('>15')) {
+            my $sp = (split(/-/, get_var('VERSION')))[-1];
+            $sle15_repo = "REPO_SLE15_" . "$sp" . "_MODULE_DEVELOPMENT_TOOLS";
+        }
+        else {
+            $sle15_repo = 'REPO_SLE15_MODULE_DEVELOPMENT_TOOLS';
+        }
+
+        my $devel_repo = get_required_var(is_sle('>=15') ? $sle15_repo : 'REPO_SLE_SDK');
+        assert_script_run "zypper ar -c $utils::OPENQA_FTP_URL/" . $devel_repo . " devel-repo";
+    }
+
+    zypper_call '--gpg-auto-import-keys ref';
+    zypper_call 'install jing';
+
+    my $rc_jing    = script_run 'jing /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml';
+    my $rc_xmllint = script_run 'xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml';
+
+    if (($rc_jing) || ($rc_xmllint)) {
```

Comment:
I suspect that jing and xmlint not just fail but give you some sane output why validation failed, which you can collect and upload as log to ease life of reviewer of this test

Reply:
It is visible in openQA. http://dhcp128.suse.cz/tests/5338#step/yast2_clone_system/19

Reply:
screenshot not the best form of storing data, you can search it, you can't copy/paste it to google find out details, also in case too big output you in danger to miss some important peace

Reply:
It's actually is a good point, which we miss a lot. I know that it was like that before, but let's output either to serial or using record_info, or use postfail_hook, as this info is required only in case of failures. Other than that LGTM

Reply:
Indeed, after exiting clone_system, the test switches to virtio console. Therefore most of the information is stored over here http://dhcp128.suse.cz/tests/5696/file/serial_terminal.txt